Formula/offlineimap.rb

```diff
@@ -24,15 +24,18 @@ def caveats; <<-EOS.undent
     EOS
   end
 
-  test do
-    system bin/"offlineimap", "--version"
-  end
+  plist_options :startup => true
```

Comment:
The only documentation I found for plist_options is [this](http://www.rubydoc.info/github/Homebrew/brew/Formula.plist_options). But it does not really explain this option for me.